CMakeLists.txt

```diff
@@ -75,6 +75,13 @@ else()
   include(DownloadFmt)
 endif()
 
+find_package(PythonInterp)
```

Comment:
uhm, this is problamtic. we're looking up python using modern cmake modules further down. you cannot reintroduce the old find module here, it will screw up everything below. Do the check after looking up python around l236